code/index.html

```diff
@@ -3,28 +3,226 @@
+        </div>
 
-    <!-- Signup form -->
+      </section>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <section class="header-image">
+        <!-- The image is located in css-file -->
+      </section>
+      
+    </header>
+   
+    <main>
+      <!-- welcome and description -->
+    <section class="welcome">
+      <h2>We are Vintage Decor</h2>
+      <p>lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut abore et dolore magna aliqua.</p>
+      <p>lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut abore et dolore magna aliqua.</p>
+      <p>lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut abore et dolore magna aliqua.</p>
+    </section>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    
+    <!-- services -->
+    <h2 class="info">What we offer</h2>
+    <section class="section-container">
+      <div class="rentals-container">
+        <img class="rentals-image" src="images/third-image-mobile.jpg" alt="wall of barn with mirror and haybale">
+        <div class="img-wrap">
+          <img class="rentals-image-desktop" src="images/first-desktop-image.jpg" alt="vintage yellow sofa">          
+          <h2 class="rentals-header">Rentals</h2>
+          <p class="rentals-text">We have beautiful vintage furnitures, porcelain and decoration for hire</p>
+        </div>
       </div>
+      <div class="event-container">
+        <img class="event-image" src= "images/fourth-mobile-image.jpg" alt="table setting">
+        <div class="img-wrap">
+          <img class="event-image-desktop" src="images/second-desktop-image.jpg" alt="table setting outdoors">
+          <div class="event-header">Events</div>
+          <div class="event-text">We have beautiful vintage furnitures, porcelain and decoration for hire </div>
+        </div>
+      </div>
+    </section>
+      
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+        <!-- get to know -->
+      <h2 class="info">Get to know us</h2>
+      <section class="section-container">
+        <div class="portfolio-container">
+          <div class="img-wrap">
+            <img class="portfolio-image" src= "images/second-image-mobile.jpg" alt="guitar decorated with flowers on chair">
+            <div class="portfolio-header">Portfolio</div>              
+            <div class="portfolio-text">Browse our previous event decorations</div>
+          </div>
+        </div>
+        <div class="about-container">
+          <div class="img-wrap">
+          <img class="about-image" src= "images/about-us.jpg" alt="2 people running on a field">
+          <div class="about-header">Our story</div>
+        </div>
+       </div>
+      </section>
+    
+    
+    <!-- Accordion -->
```

Comment:
You accordion is wonderful with the picture as background and the transparent answers! Great job! 

(A very small detail, but there is a bit of an overflow when all answers are expanded - unfortunately I do not have any code suggestions for fixing it.)